docusaurus/docs/learn/quickstarts/network/miniziti.bash

```diff
@@ -5,14 +5,17 @@ set -o nounset
 set -o pipefail
 
 banner(){
+
+    local profile="${1-}"
+
     cat <<BANNER
                   __   ___
     |\/| | |\ | |  / |  |  |
     |  | | | \| | /_ |  |  |
 
 BANNER
 
-    if (( $# )); then
+    if [[ -n "$profile" ]] && [[ "$profile" != "miniziti" ]]; then
```

Comment:
I have a slight preference for compound expressions over multiple invocations of `[[` or `test`.

```suggestion
    if [[ -n "$profile" && "$profile" != "miniziti" ]]; then
```

Reply:
Can update.

Reply:
Should be resolved.